specs/002-directus-cms-setup/data-model.md

```
@@ -0,0 +1,262 @@
+# Data Model: Directus CMS Setup
+
+**Feature**: 002-directus-cms-setup | **Date**: 2025-12-05
+**Source**: `directus-schema/schema.json`
+
+## Entity Relationship Diagram
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)